### ISSUE TITLE
change needed to src/Makefile.am to complete install on Ubuntu 15.04

### ISSUE THREAD

**Issue by xxdesmus** [gamelinux/passivedns#50]
Following the install instructions when I get here ```autoreconf --install``` I get the following:

```
~/passivedns# autoreconf --install
configure.ac:6: installing './compile'
configure.ac:211: installing './config.guess'
configure.ac:211: installing './config.sub'
configure.ac:5: installing './install-sh'
configure.ac:5: installing './missing'
src/Makefile.am:12: warning: deprecated feature: target 'passivedns' overrides 'passivedns$(EXEEXT)'
src/Makefile.am:12: change your target to read 'passivedns$(EXEEXT)'
/usr/share/automake-1.14/am/program.am: target 'passivedns$(EXEEXT)' was defined here
src/Makefile.am:5:   while processing program 'passivedns'
src/Makefile.am: installing './depcomp'
```
I followed the output and made this change to ```src/MakeFile.am``` on line 12, replacing ```passivedns``` with ```passivedns$(EXEEXT)```  and then  ```autoreconf --install```  completed without issue.

**Comment by BobuSumisu** [comment]
Same error on Ubuntu 14.04.

    autoreconf (GNU Autoconf) 2.69
    automake (GNU automake) 1.14.1

**Comment by DigiAngel** [comment]
Ya here too on 14.04...the fix works well.

**Comment by thus** [comment]
This was fixed in #51, so closing the issue.
